Makefile

```diff
@@ -4,7 +4,7 @@
 .DEFAULT_GOAL := default
 
 cwd := $(dir $(realpath $(firstword $(MAKEFILE_LIST))))
-sagefile := $(cwd)/.sage/bin/sagefile
+sagefile := $(realpath $(cwd)/.sage/bin/sagefile)
```

Comment:
Do we need to add this below aswell, where we define the `go` variable?

Reply:
I'll add it as well, and I think I was able to test it properly in a `node:latest` container.